docs/README.md

```diff
@@ -1,36 +1,49 @@
+# PowSyBl OpenLoadflow documentation
+
 These are the documentation sources for the powsybl-open-loadflow features.
 
 Please keep them up to date with your developments.  
 They are published on [powsybl-open-loadflow.readthedocs.io](http://powsybl-open-loadflow.readthedocs.io/) and pull requests are built and previewed automatically.
 
+## Build the documentation
+
 When modifying the website content, you can easily preview the result on your PC.
 
-**First option - in a terminal, navigate to the root of the project and run the following commands:**
+### From a terminal
 
+Navigate to the root of the project and run the following commands:
+
+Install the requirements the first time:
+~~~bash
+pip install -r requirements.txt
 ~~~
-pip install -r docs/requirements.txt
-sphinx-build -a docs ./build-docs
+Build the documentation:
+~~~bash
+sphinx-build -a . ./_build/html
 ~~~
-
-*Note: if the build fails, try with the `-E` option to clear the cache:*
-
+Or
+~~~bash
+make html
 ~~~
-sphinx-build -a -E docs ./build-docs
+Or to build in latex format
+~~~bash
+make latexpdf
 ~~~
 
-**Second option - run the following commands directly from your IDE GUI**
+### From your IDE GUI
```

Comment:
As @flo-dup noted on the `powsybl-core` equivalent PR (powsybl/powsybl-core#3311):

> This is not needed anymore as the commands are now the same